internal/scorecard/alpha/bundle.go

```diff
@@ -18,20 +18,21 @@ import (
 	"fmt"
 	"io/ioutil"
 	"os"
+	"path/filepath"
 
 	"k8s.io/apimachinery/pkg/util/rand"
 )
 
 // getBundleData tars up the contents of a bundle from a path, and returns that tar file in []byte
-func (o Scorecard) getBundleData() (bundleData []byte, err error) {
+func (o Scorecard) GetBundleData() (bundleData []byte, err error) {
 
 	// make sure the bundle exists on disk
 	_, err = os.Stat(o.BundlePath)
 	if os.IsNotExist(err) {
 		return bundleData, fmt.Errorf("bundle path is not valid %w", err)
 	}
 
-	tempTarFileName := fmt.Sprintf("%s%ctempBundle-%s.tar", os.TempDir(), os.PathSeparator, rand.String(4))
+	tempTarFileName := filepath.Join(os.TempDir(), fmt.Sprintf("tempBundle-%s.tar.gz", rand.String(4)))
 
 	paths := []string{o.BundlePath}
 	err = CreateTarFile(tempTarFileName, paths)
```

Comment:
For follow-on: We should contruct the tar.gz data in memory rather than using temporary files.

Reply:
I'll mark this as a TODO for another PR.... do you want me to use spf13/afero or some other implementation?

Reply:
I was thinking of chaining the tar and gzip writers together and using `io.Copy` to copy from a file handle into the composite tar/gzip writer. 

Pretty much this: https://medium.com/@skdomino/taring-untaring-files-in-go-6b07cf56bc07